src/Microsoft.Azure.WebJobs.Host/Scale/ScaleStatus.cs

```diff
@@ -12,5 +12,21 @@ public class ScaleStatus
         /// Gets or sets the current scale decision.
         /// </summary>
         public ScaleVote Vote { get; set; }
+
+        /// <summary>
+        /// Gets or sets the currebt target worker count.
+        /// </summary>
+        public int? TargetWorkerCount { get; set; }
+
+        /// <summary>
+        /// Gets the current scale decision as int.
+        /// </summary>
+        public int VoteAsInt
```

Comment:
This int value makes for not a clean Type interface. We should just use ScaleVote in the SDK, and if SC needs to do a conversion/interpretation, it should do it itself in that layer.

Reply:
Ok, this can be moved to SC.